lib/diagnostic.rb

```diff
@@ -115,10 +115,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = %w(first_name: #{first_name} age: #{age} )'
 
```

Comment:
Close, but `%w()` is for creating arrays, not hashes.

`person = {first_name: "Natasha", age: 88}`